packages/dev/gui/src/3D/controls/touchHolographicButton.ts

```diff
@@ -391,6 +399,14 @@ export class TouchHolographicButton extends TouchButton3D {
         this.collisionMesh = collisionMesh;
         this.collidableFrontDirection = this._backPlate.forward.negate(); // Mesh is facing the wrong way
 
+        this._pointerObserver = scene.onPointerObservable.add((pointerInfo) => {
+
+            if (this.isActiveNearInteraction && pointerInfo.pickInfo) {
+                if (pointerInfo.pickInfo.hit)
+                    this._isRay = (<PointerEvent><unknown>pointerInfo.event).pointerType === "xr";

```

Comment:
this is for general XR and not near interaction, right?

Reply:
Yes, I believe so. Previously I used 
```typescript
if ((<PointerEvent>pointerInfo.event).pointerType === "xr") {
    this._isRay = true;
}
if ((<PointerEvent>pointerInfo.event).pointerType === "xr-near") {
    this._isRay = false;
}
```
but since I really only want to know if it is an XR ray interaction going on I simplified it to the code committed in the PR.

So basically I wanted a way to differentiate between XR ray interactions and XR near interactions, but here's where my knowledge regarding the differentiation between XR and XR near interaction stops, so if you have any recommendations or think this doesn't fit with the XR/XR near interaction architecture, let me know and I can continue to debug it or change behavior based on your expertise. 

The proposed fix I committed is imo a bit hacky, so if there's any more elegant way to resolve the bug I'm more than happy to implement it with some pointers on where to look! 😊